src/emailservice/email_server.py

```diff
@@ -185,13 +190,7 @@ def initStackdriverProfiling():
       raise KeyError()
     else:
       logger.info("Tracing enabled.")
-      sampler = always_on.AlwaysOnSampler()
-      exporter = stackdriver_exporter.StackdriverExporter(
-        project_id=os.environ.get('GCP_PROJECT_ID'),
-        transport=AsyncTransport)
-      tracer_interceptor = server_interceptor.OpenCensusServerInterceptor(sampler, exporter)
```

Comment:
On line 189 there is an exporter setup to export traces to stackdriver. I don't see in your code where do we setup exporting traces

Reply:
resolving because outdated, i definitely have an exporter in the new implementation :)